modules/apps/account/account-service/src/main/java/com/liferay/account/internal/instance/lifecycle/AddDefaultAccountRolesPortalInstanceLifecycleListener.java

```diff
@@ -129,20 +140,76 @@ private void _addRole(long userId, String roleName) throws PortalException {
 			HashMapBuilder.put(
 				LocaleThreadLocal.getDefaultLocale(), roleName
 			).build(),
-			null, RoleConstants.TYPE_REGULAR, null, null);
+			_getDescriptionMap(roleName), RoleConstants.TYPE_REGULAR, null,
+			null);
 	}
 
 	private boolean _exists(String roleName) {
 		Role role = _roleLocalService.fetchRole(
 			CompanyThreadLocal.getCompanyId(), roleName);
 
 		if (role != null) {
+			Map<Locale, String> descriptionMap = _getDescriptionMap(roleName);
+
+			if (!Objects.equals(role.getDescriptionMap(), descriptionMap)) {
+				role.setDescriptionMap(descriptionMap);
+
+				_roleLocalService.updateRole(role);
+			}
+
 			return true;
 		}
 
 		return false;
 	}
 
+	private String _getDescription(Locale locale, String roleName) {
+		ResourceBundle resourceBundle =
+			_resourceBundleLoader.loadResourceBundle(locale);
+
+		if (Objects.equals(
+				AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_ADMINISTRATOR,
+				roleName)) {
+
+			return ResourceBundleUtil.getString(
+				resourceBundle,
+				"account-administrators-are-super-users-of-their-account");
+		}
+		else if (Objects.equals(
+					AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MANAGER,
+					roleName)) {
+
+			return ResourceBundleUtil.getString(
+				resourceBundle,
+				"account-managers-who-belong-to-an-organization-can-" +
+					"administer-all-accounts-associated-to-that-organization");
+		}
+		else if (Objects.equals(
+					AccountRoleConstants.REQUIRED_ROLE_NAME_ACCOUNT_MEMBER,
+					roleName)) {
+
+			return ResourceBundleUtil.getString(
+				resourceBundle,
+				"all-users-who-belong-to-an-account-have-this-role-within-" +
+					"that-account");
+		}
+
+		return StringPool.BLANK;
+	}
+
+	private Map<Locale, String> _getDescriptionMap(String roleName) {
+		Set<Locale> availableLocales = _language.getAvailableLocales();
+
+		Stream<Locale> stream = availableLocales.stream();
+
+		return stream.map(
+			locale -> new AbstractMap.SimpleEntry<>(
+				locale, _getDescription(locale, roleName))
+		).collect(
+			Collectors.toMap(Map.Entry::getKey, Map.Entry::getValue)
+		);
```

Comment:
We can inline `_getDescription` into this this method by using `ResourceBundleUtil.getLocalizationMap`. It will iterate the available locales for us! 🥳

Reply:
Oh nice, I didn't know that. Will update the code to use `ResourceBundleUtil`